packages/contracts-bedrock/contracts/L1/L1StandardBridge.sol

```diff
@@ -240,10 +218,32 @@ contract L1StandardBridge is StandardBridge, Semver {
         address _to,
         uint256 _amount,
         bytes calldata _extraData
-    ) external payable onlyOtherBridge {
+    ) external payable {
         finalizeBridgeETH(_from, _to, _amount, _extraData);
```

Comment:
Confirming that the `onlyOtherBridge` modifier is here: https://github.com/ethereum-optimism/optimism/blob/5101faaf61b3e9303a71d80dd59f4a145f6de760/packages/contracts-bedrock/contracts/universal/StandardBridge.sol#L299